integration/entity.go

```diff
@@ -79,8 +95,8 @@ func (e *Entity) AddEvent(event *metric.Event) error {
 	return nil
 }
 
-// AddInventory method adds a inventory item.
-func (e *Entity) AddInventory(key string, field string, value interface{}) {
+// SetInventoryItem method adds a inventory item.
+func (e *Entity) SetInventoryItem(key string, field string, value interface{}) {
```

Comment:
Maybe `SetInventory` is shorter and equally meaningful (since _Item_ is very generic)

Reply:
@mariomac This was discussed offline and we all agreed on set is as `SetInventoryItem` because Inventory is the whole Inventory, and what we are doing here is setting an item in the Inventory map.